Auto attach executables to release

da-kami opened the pull request:
Release manually triggered in Github UI.
Executables are built and uploaded to the release by the workflow.

da-kami committed:
Add attaching build artifacts to releases

Release manually triggered in Github UI.
Executables are built and uploaded to the release by the workflow.

da-kami committed:
Cleanup package workflow

D4nte commented:
Did you try the workflow on your fork?

thomaseizinger commented on .github/workflows/package.yml:
```diff
@@ -60,10 +59,8 @@ jobs:
       #       yarn test-e2e
```
Could delete all this commented code!

da-kami committed:
Set application types according to operating system

For Linux and Max there is no official mime type registered, thus set to octet-stream.

da-kami commented:
> Did you try the workflow on your fork?

Since it requires Botty MacBotface I will just try it directly on Ambrosia. (Otherwise we have to share tokens, ...).
Nobody is using Ambrosia yet, so no worries trying it out.

da-kami committed:
Re-add GH token to package step after error on action run
